docs/content/about/tutorial.md

```diff
@@ -169,41 +172,82 @@ nikita.call({
+}, function({config}) {
   console.info(config.who)
 })
 ```
 
+### Parent action and children
+
+The Nikita session is organized as a hierarchical tree of actions. The parent is an action of the higher level in the session tree. All Nikita's actions have a parent action except the root Nikita action instantiating a Nikita session. The children are the actions executed in the handler of a parent action using the [`this` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this):
+
+```js
+// Root action
+nikita(function() {
+  console.info('I am root')
+})
+// Parent action
+.call(function() {
+  console.info('I am parent')
+  // Child action
+  this.call(function() {
+    console.info('I am child')
+  })
+})
+```
+
+Alternatively, you can use the [JavaScript arrow function expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) reduces the syntax. In such a case, to call a child action you must use the [`context` property](/current/action/context), because arrow functions don't provide their own `this` binding. `context` is available inside the handler as a property of the first argument:
+
+```js
+// Root action
+nikita(() => {
+  console.info('I am root')
+})
+// Parent action
+.call(({context}) => {
+  console.info('I am parent')
+  // Child action
+  context.call(() => {
+    console.info('I am child')
+  })
+})
+```
+
 ### Action promise
 
-Nikita's actions always return [Javascript Promises](https://nodejs.dev/learn/understanding-javascript-promises). To access the action output, you have to use the Promise API or simply use the `async/await` construct to get the returned value.
+Nikita's actions always return [Javascript Promises](https://nodejs.dev/learn/understanding-javascript-promises). To access the [action output](/current/action/output), you have to use the Promise API or simply use the [`async/await` construction](https://nodejs.dev/learn/modern-asynchronous-javascript-with-async-and-await) to get the returned value.
 
 ```js
 // Call asynchronous function
 (async () => {
-  // Await result from Promise
-  var result = await nikita.call({
-    who: 'leon',
-  }, ({config}) => {
-    return {who: config.who}
-  })
+  // Define empty array
+  history = []
+  // Access the action output
+  const result = await nikita.call(() => 'first')
   // Run next synchronous function
-  console.info(result.who)  // leon
+  history.push(result)
+  // Print array
+  console.info(history) // [ 'first' ]
 })()
 ```
 
-Nikita also provides you the guarantee that your actions are executed sequentially:
+Nikita also provides you the guarantee that your actions are executed sequentially. The following example is calling 3 actions, where 2 of them are executing asynchronous functions in the handlers. The sequential execution is asserted by the `history` array containing the values according to the actions' calls and the `result` value returned by the 3rd action:
```

Comment:
The following example is calling 3 asynchronuous actions of various duration. The history array records their order of execution: